pkg/consts/enrichment_injection.go

```diff
@@ -2,12 +2,14 @@ package consts
 
 const (
 	EnrichmentEndpointSecretName = "dynatrace-metadata-enrichment-endpoint"
-	EnrichmentFilenameTemplate   = "dt_metadata.%s"
 	EnrichmentInjectedEnv        = "METADATA_ENRICHMENT_INJECTED"
 	EnrichmentWorkloadKindEnv    = "DT_WORKLOAD_KIND"
 	EnrichmentWorkloadNameEnv    = "DT_WORKLOAD_NAME"
+	EnrichmentPropertiesFilename = "dt_metadata.properties"
+	EnrichmentJsonFilename       = "dt_metadata.json"
 )
 
 var (
 	EnrichmentMountPath = "/var/lib/dynatrace/enrichment"
+	EnrichmentInitPath  = "/tmp/enrichment"
```

Comment:
why aren't these `const`?

Reply:
Not sure, did not want to change them, but IMHO they should be consts, good point

Reply:
done